concrete/src/Entity/User/GroupSignupRequestAccept.php

```diff
@@ -15,170 +19,165 @@
 class GroupSignupRequestAccept implements SubjectInterface
 {
     /**
-     * @var int
      * @ORM\Id
      * @ORM\Column(type="integer", options={"unsigned":true})
      * @ORM\GeneratedValue(strategy="AUTO")
+     *
+     * @var int|null NULL if not yet flushed to the database
      */
     protected $id;
 
     /**
      * @ORM\OneToMany(targetEntity="\Concrete\Core\Entity\Notification\GroupSignupRequestAcceptNotification", mappedBy="signup", cascade={"remove"}),
+     *
+     * @var \Doctrine\Common\Collections\Collection
      */
     protected $notifications;
 
     /**
-     * @var int
      * @ORM\Column(type="integer", options={"unsigned":true})
+     *
+     * @var int
      */
     protected $gID;
 
     /**
-     * @var User
      * @ORM\ManyToOne(targetEntity="\Concrete\Core\Entity\User\User")
      * @ORM\JoinColumn(name="uID", referencedColumnName="uID", onDelete="SET NULL")
+     *
+     * @var \Concrete\Core\Entity\User\User|null
      */
     protected $user;
 
     /**
-     * @var User
      * @ORM\ManyToOne(targetEntity="\Concrete\Core\Entity\User\User")
      * @ORM\JoinColumn(name="managerUID", referencedColumnName="uID", onDelete="SET NULL")
+     *
+     * @var \Concrete\Core\Entity\User\User|null
      */
     protected $manager;
 
     /**
-     * @var \DateTime
      * @ORM\Column(type="datetime")
+     *
+     * @var \DateTime
      */
-    protected $RequestAccepted = null;
+    protected $RequestAccepted;
 
     /**
-     * GroupSignupRequestAccept constructor.
-     * @param Group $group
-     * @param \Concrete\Core\User\User $user
-     * @param \Concrete\Core\User\User $manager
-     * @throws \Exception
+     * @param \Concrete\Core\User\Group\Group|null $group
+     * @param \Concrete\Core\User\User|null $user
+     * @param \Concrete\Core\User\User|null $manager
      */
     public function __construct($group = null, $user = null, $manager = null)
     {
+        $this->notifications = new ArrayCollection();
         if ($group instanceof Group) {
             $this->gID = $group->getGroupID();
         }
-
         if ($user instanceof \Concrete\Core\User\User) {
-            if ($user->getUserInfoObject() instanceof UserInfo) {
-                $this->user = $user->getUserInfoObject()->getEntityObject();
+            $userInfo = $user->getUserInfoObject();
+            if ($userInfo instanceof UserInfo) {
+                $this->user = $userInfo->getEntityObject();
             }
         }
-
         if ($manager instanceof \Concrete\Core\User\User) {
-            $this->manager = $user->getUserInfoObject()->getEntityObject();
```

Comment:
There was a bug here: the `manager` field was initialized starting from the `$user` parameter: it should be `$manager`, isn't it?